src/ECER.Managers.Admin/InviteLinkHandlers.cs

```diff
@@ -23,7 +23,7 @@ public async Task<VerifyInviteTokenCommandResponse> Handle(VerifyInviteTokenComm
   {
```

Comment:
This is where you want to implement the use case - you may want to reconsider the name of the command